content/Developement/quickStart.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "QuickStarted"
+date: 2023-05-12T17:51:06+05:30
+draft: false
+---
+
+### Introduction
+
+---
+Run Apache Hive inside docker container in pseudo-distributed mode, provide the following
+- Quick-start/Debugging/Prepare a test env for Hive
+
+
+### Quickstart
+
+---
+
+##### Step 1:  Pull the image
+DockerHub: https://hub.docker.com/r/apache/hive/tags
+ 
+Currently, there are 3 images released:
+ - 4.0.0-alpha-2
+ - 4.0.0-alpha-1
+ - 3.1.3
+```shell
+docker pull apache/hive:4.0.0-alpha-2
+```
+
+##### STEP 2: Export the Hive version 
+```shell
+export HIVE_VERSION=<Hive Tag/Image Version>
+```
+
+Example:
+```shell
+export HIVE_VERSION=4.0.0-alpha-2
+```
+
+##### STEP 3:  Launch the HiveServer2 with an embedded Metastore.
+
+This is lightweight and for a quick setup, it uses Derby as metastore db.
+```shell
+docker run -d -p 10000:10000 -p 10002:10002 --env SERVICE_NAME=hiveserver2 --name hive4 apache/hive:${HIVE_VERSION}
+```
+##### STEP 4: Connect to beeline
+
+```shell
+docker exec -it hiveserver2 beeline -u 'jdbc:hive2://hiveserver2:10000/'
+```
+
+##### Note: Launch Standalone Metastore
+To use standalone Metastore with Derby,
+
+```shell
+docker run -d -p 9083:9083 --env SERVICE_NAME=metastore --name metastore-standalone apache/hive:${HIVE_VERSION}
+```
+
+## Detailed Setup
+
+---
+
+#### Build image
+Apache Hive relies on Hadoop, Tez and some others to facilitate reading, writing, and managing large datasets.
+The `build.sh` provides ways to build the image against specified version of the dependent, as well as build from source.
```

Comment:
nit: should we provide a relative path to the `build.sh`

Reply:
Done.